run-tests.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+# Runs DDQ backend functional tests.
+#
+# Examples
+#
+#   run-tests.sh
+#
+# Returns nothing.
+set -e
+thisDir="${0%/*}"
+
+# Running all tests rather then exiting if one failed.
```

Comment:
The success/failure status code won't be returned for the first two tests.  How would a CI system catch problems?